File: eBordo.Model/Models/IgracSkills.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eBordo.Model.Models
{
    public class IgracSkills
    {
        public int? igracSkillsId { get; set; }

        public float kontrolaLopte { get; set; }
        public int kontrolaLopteZbir { get; set; }

        public float driblanje { get; set; }
        public int dribljanjeZbir { get; set; }

        public float dodavanje { get; set; }
        public int dodavanjeZbir { get; set; }

        public float kretanje { get; set; }
        public int kretanjeZbir { get; set; }

        public float brzina { get; set; }
        public int brzinaZbir { get; set; }

        public float sut { get; set; }
        public int sutZbir { get; set; }

        public float snaga { get; set; }
        public int snagaZbir { get; set; }

        public float markiranje { get; set; }
        public int markiranjeZbir { get; set; }

        public float klizeciStart { get; set; }
        public int klizeciStartZbir { get; set; }

        public float skok { get; set; }
        public int skokZbir { get; set; }

        public float odbrana { get; set; }
        public int odbranaZbir { get; set; }

        public float zbirOcjena { get; set; }
        public float prosjekOcjena { get; set; }
    }
}

File: ebordo_mobile/lib/models/igrac-skills.dart
class IgracSkills {
  final num kontrolaLopte;
  final num driblanje;
  final num dodavanje;
  final num kretanje;
  final num brzina;
  final num sut;
  final num snaga;
  final num markiranje;
  final num klizeciStart;
  final num skok;
  final num odbrana;
  final num prosjekOcjena;

  IgracSkills({
    required this.kontrolaLopte,
    required this.driblanje,
    required this.dodavanje,
    required this.kretanje,
    required this.brzina,
    required this.sut,
    required this.snaga,
    required this.markiranje,
    required this.klizeciStart,
    required this.skok,
    required this.odbrana,
    required this.prosjekOcjena,
  });

  factory IgracSkills.fromJson(Map<String, dynamic> json) {
    return IgracSkills(
      kontrolaLopte: json["kontrolaLopte"],
      driblanje: json["driblanje"],
      dodavanje: json["dodavanje"],
      kretanje: json["kretanje"],
      brzina: json["brzina"],
      sut: json["sut"],
      snaga: json["snaga"],
      markiranje: json["markiranje"],
      klizeciStart: json["klizeciStart"],
      skok: json["skok"],
      odbrana: json["odbrana"],
      prosjekOcjena: json["prosjekOcjena"],
    );
  }

  Map<String, dynamic> toJson() => {
        "kontrolaLopte": kontrolaLopte,
        "driblanje": driblanje,
        "dodavanje": dodavanje,
        "kretanje": kretanje,
        "brzina": brzina,
        "sut": sut,
        "snaga": snaga,
        "markiranje": markiranje,
        "klizeciStart": klizeciStart,
        "skok": skok,
        "odbrana": odbrana,
        "prosjekOcjena": prosjekOcjena
      };
}
